gamemode/config/crafting.lua

```diff
@@ -29,6 +29,37 @@ recipe = {
 ]]
 
 config.RECIPES = {
+    {
+        name = "Water",
+        icon = "materials/rvr/items/water_bottle.png",
+        crafterType = "water",
+        recipes = {
+            {
+                item = "water",
+                count = 1,
+                timeToCraft = 39,
+                ingredients = {
+                    dirty_water =  1
+                }
+            }
+        }
+    },
```

Comment:
Is this just craftable all the time? Or is it only craftable via the water purifier?

Reply:
Also, where is the recipe for the purifier?